src/components/ProductDetail/ProductDetailOrderForm.js

```diff
@@ -0,0 +1,141 @@
+import { useEffect, useState } from "react";
+import SizeOptionsContainer from "./SizeOptionsContainer";
+import { camelToPascalCase } from "../../helperFunctions";
+import { Link } from "react-router-dom";
+
+const ProductDetailOrderForm = ({product, addToShoppingBag, shoppingBag, updateQuantity}) => {
+  const [isSingleSize, setIsSingleSize] = useState(false);
+  const [inputFields, setInputFields] = useState({
+    "color":"",
+    "size": "",
+    "quantity":0
+  });
+  const [colorOptions, setColorOptions] = useState([]);
+  const [isFormHealthy, setIsFormHealthy] = useState(false);
+  const [totalQuantity, setTotalQuantity] = useState(0);
+  
+  useEffect(() => {
+    setColorOptions(product?.colorOptions)
+    console.log(product)
```

Comment:
Lots of console logs, could remove before merge?

Reply:
uh sorry. Removed most but a couple left. Will take care of those.